src/python/beta_to_450k.py

```diff
@@ -90,7 +90,8 @@ def betas2csv(args):
     # set reference sites, as the intersection of the user input (--ref)
     # and the "full" reference, supplied by wgbstools (ilmn2cpg_dict)
     df = read_reference(args)
-    indices = np.array(df['cpg'])
+    df = df.dropna(how='any')
```

Comment:
please move this code to be inside the `load_full_ref` method. Perhaps line 42. Also please put the code inside an `if` block which checks if the reference is hg38 since this issue only exists with the hg38 reference file.